jsc-android-manager/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "jsc-android-manager",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "jsc-android": "^216113.0.0"
```

Comment:
Maybe lock this version by remove `^` to keep it predictable.

Reply:
Any special reason not put this deps in root `package.json`

Reply:
Version was locked and updated to latest from [npm/jsc-android]. Test plan succeeded. 🤞 on ci tests